sdk/identity/azure-identity/tests/helpers.py

```diff
@@ -137,6 +137,10 @@ def async_validating_transport(requests, responses):
         sync_transport = validating_transport(requests, responses)
         return mock.Mock(send=asyncio.coroutine(sync_transport.send))
 
+    def async_return(result):
```

Comment:
Please rebase onto master, [this exists there already](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/identity/azure-identity/tests/helpers_async.py#L13).

Reply:
ported related code from master branch to this branch